docs/pages/enroll-resources/kubernetes-access/health-checks.mdx

```diff
@@ -0,0 +1,276 @@
+---
+title: Teleport Kubernetes Health Checks
+sidebar_label: Health Checks
+description: How to configure Teleport Kubernetes health checks and view health.
+tags:
+- conceptual
+- zero-trust
+- infrastructure-identity
+---
+
+This documentation provides an overview of health checks to Kubernetes clusters with Teleport. Teleport Kubernetes Service instances periodically check the connectivity and permissions of enrolled Kubernetes clusters, and is available in Teleport version (= kubernetes.health_check_min_version =) and later.
+
+## Why check the health of a Kubernetes cluster?
+
+- **Observability**: Discover network and permission issues before users do. Unhealthy Kubernetes clusters are visible in a Teleport UI, command line tool, or Prometheus metrics.
+- **High Availability**: Automatically route and distribute connections to healthy Kubernetes clusters in a high-availability configuration.
+
+## What's checked?
+
+Kubernetes permissions and a Kubernetes health endpoint are checked to determine whether a Kubernetes cluster is both up and usable with Teleport.
```

Comment:
Since this is a conceptual and architectural guide, I would replace passive-voice constructions like "are checked" with active ones, e.g., "The Teleport Kubernetes Service checks...". This way, it's clear what Teleport components are responsible for what.